tests/utils/blueprints/process-template/escape.test.js

```diff
@@ -0,0 +1,16 @@
+import { processTemplate } from '../../../../src/utils/blueprints.js';
+import { assert, test } from '../../../helpers/testing.js';
+
+test('utils | blueprints | process-template > escape', function () {
+  const blueprintFile = '<%- context.htmlCode %>';
```

Comment:
At the moment, none of the blueprints make use of the `escape` feature from `lodash.template`.